Source/Urho3D/Input/Input.cpp

```diff
@@ -322,11 +322,13 @@ int Win32_ResizingEventWatcher(void* data, SDL_Event* event)
                     if (graphics->IsInitialized())
                     {
                         graphics->OnWindowResized();
+#ifndef UWP
                         if (auto* engine = ctx->GetSubsystem<Engine>())
                         {
                             if (engine->IsInitialized())
                                 engine->RunFrame();
                         }
+#endif
```

Comment:
Is there a point in having `Win32_ResizingEventWatcher` run at all on uwp? I already disabled it on master, but you did it differently. Not sure which version to keep.